apitest/scripts/trade-simulation-env.sh

```diff
@@ -0,0 +1,219 @@
+#! /bin/bash
+
+# This file must be sourced by the main driver.
+
+export CLI_BASE="./bisq-cli --password=xyz"
+export ARBITRATOR_PORT=9997
+export ALICE_PORT=9998
+export BOB_PORT=9999
+export F2F_ACCT_FORM="f2f-acct.json"
```

Comment:
Fixed in commit 8b4938a9e.

Reply:
Turns out codacy requires these vars be exported, or it fails the codacy analysis, and cannot be merged.  It may be fixed by tweaking the codacy config, but that won't happen immediately.